Cargo.toml

```diff
@@ -22,6 +22,7 @@ dump_ivf = ["ivf"]
 quick_test = []
 desync_finder = []
 bench = []
+check_asm = []
```

Comment:
If you're going to go for checkasm style testing rather than unit tests, you need to turn checkasm on for ci.